assembl/models/idea_graph_view.py

```diff
@@ -135,25 +109,13 @@ def get_discussion_conditions(cls, discussion_id, alias_maker=None):
         return ((cls.sub_graph_id == ExplicitSubGraphView.id),
                 (ExplicitSubGraphView.discussion_id == discussion_id))
 
-    discussion = relationship(
```

Comment:
does it not break anything to remove the whole relationship instead of just the info parameter?

Reply:
It was a bad remove, thanks for the spot!

Reply:
Then re-add it?

Reply:
I did. (invisible rebase, but it's there.)

Reply:
wow strange, even the https://github.com/assembl/assembl/pull/381/files page shows it as removed

Reply:
??? Not for me... maybe reload?

Reply:
ah! yes, there's a problem.